lib/config/options/index.ts

```diff
@@ -1555,6 +1555,14 @@ const options: RenovateOptions[] = [
     allowedValues: ['strict', 'flexible', 'none'],
     default: 'strict',
   },
+  {
+    name: 'prAllowMaintainerEdits',
+    description: 'Decides if maintainers can edit Renovate pull requests.',
+    type: 'boolean',
+    supportedPlatforms: ['github'],
+    default: true,
+    globalOnly: true,
```

Comment:
so only the bot owner can change it? how would this PR help with forking renovate app?

Reply:
good point. should we go back to make it a repository setting?

Reply:
I've switched it back to a per-repo setting since the forking renovate GitHub app is a good argument. Since you also want this to be configurable, this seems like the only option.